sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -205,8 +205,8 @@ export interface BlobAcquireLeaseOptions extends CommonOptions {
 // @public
 export class BlobBatch {
     constructor();
-    deleteBlob(blobClient: BlobClient, options?: BlobDeleteOptions): Promise<void>;
     deleteBlob(url: string, credential: SharedKeyCredential | AnonymousCredential | TokenCredential, options?: BlobDeleteOptions): Promise<void>;
+    deleteBlob(blobClient: BlobClient, options?: BlobDeleteOptions): Promise<void>;
```

Comment:
These diffs are caused by different api-extractor output in Node 12. Note that it's just a different ordering, no actual API change.